internal/httputil/response.go

```diff
@@ -24,9 +24,10 @@ func InternalServerError(w http.ResponseWriter, _ *http.Request) {
 	http.Error(w, http.StatusText(status), status)
 }
 
-func RemoveResponseTransferEncoding(response *http.Response, encoding string) {
+// RemoveResponseTransferEncoding will remove a transfer encoding hint from the response.
```

Comment:
is this related?

Reply:
nope, just coverage, ignore it (test only change)